src/tonic/mod.rs

```diff
@@ -44,13 +46,15 @@ pub fn init(toc: Arc<TableOfContent>, host: String, grpc_port: u16) -> std::io::
             let service = QdrantService::default();
             let collections_service = CollectionsService::new(toc.clone());
             let points_service = PointsService::new(toc.clone());
+            let points_internal_service = PointsInternalService::new(toc.clone());
 
             log::info!("Qdrant gRPC listening on {}", grpc_port);
 
             Server::builder()
                 .add_service(QdrantServer::new(service))
                 .add_service(CollectionsServer::new(collections_service))
                 .add_service(PointsServer::new(points_service))
+                .add_service(PointsInternalServer::new(points_internal_service)) // TODO serve from different port
```

Comment:
Maybe an issue for this?

Reply:
Good point :+1: 

https://github.com/qdrant/qdrant/issues/430